src/luminance.js

```diff
@@ -6,10 +6,12 @@ import set from "./set.js";
 import xyz_d65 from "./spaces/xyz-d65.js";
 
 export function getLuminance (color) {
+	// Assume getColor() is called on color in get()
```

Comment:
I don't think this is true yet in this PR? I adjusted in https://github.com/lloydk/color.js/pull/1

Reply:
It was done in #456. Should I rebase this PR?

Reply:
Oh, yeah I missed that, good call. Sure, could you rebase?